src/Microsoft.AspNetCore.Server.Kestrel.Core/KestrelServerLimits.cs

```diff
@@ -134,6 +139,28 @@ public int MaxRequestHeaderCount
         }
 
         /// <summary>
+        /// Gets or sets the maximum allowed size of any request body in bytes.
+        /// When set to null, the maximum request body size is unlimited.
+        /// This limit has no affect on upgraded connections which are always unlimited.
+        /// This can be overridden by <see cref="IHttpMaxRequestBodySizeFeature"/>.
```

Comment:
I would change this to "This can be overridden per-request via <see cref="IHttpMaxRequestBodySizeFeature"/>."